docker/jenkins/push2dockerhub.sh

```diff
@@ -20,8 +20,17 @@ docker push $DOCKER_REPOSITORY:$COMMIT
 
 GIT_TAG="$(git describe --tags --exact-match $GIT_COMMIT 2> /dev/null)"
 if [[ ! -z $GIT_TAG ]]; then
+    # tag and push docker image with git tag
     docker tag -f $FROM_DOCKER_REPOSITORY:$COMMIT $DOCKER_REPOSITORY:$GIT_TAG
     docker push $DOCKER_REPOSITORY:$GIT_TAG
+
+    # tag and push docker image with "latest"
     docker tag -f $FROM_DOCKER_REPOSITORY:$COMMIT $DOCKER_REPOSITORY:latest
     docker push $DOCKER_REPOSITORY:latest
+
+    # tag and push base image with latest
+    BASE_IMAGE="$(docker run mozorg/bedrock bash -c "head -n 1 Dockerfile" | sed "s/FROM //")"
```

Comment:
So there are two jenkins jobs that run the `push2dockerhub` script

* https://ci.us-west.moz.works/view/Bedrock/job/bedrock_push_dockerhub/configure
* https://ci.us-west.moz.works/view/Bedrock/job/bedrock_code_push_dockerhub/configure

Here you get the `FROM` of `mozorg/bedrock` which should be `mozorg/bedrock_code` but it's `bedrock_base` b/c the image squashing is not working the way it's supposed to work, but that's another bug.

I'm wondering whether it's better to implement another jenkins job to `bedrock_base_push_dockerhub`?